src/presentation/pages/Home/MyPage/NeogaPick/index.tsx

```diff
@@ -23,19 +51,28 @@ function MyNeogaPick() {
           너가소개서에 지인이 남겨준 답변들 중<br />
           <span>My 프로필에 걸어두고 싶은 답변</span>을 <span>픽</span>해주세요!
         </header>
-        {isNSSBookmarkLoading ? (
-          <div>너가소개서 픽 하기 정보 로딩중</div>
-        ) : (
-          neososeoBookmark && (
-            <StMyNeogaPickList>
-              {neososeoBookmark.count > 0 ? (
-                <NeososeoAnswerCardList answers={neososeoBookmark.answerList} />
-              ) : (
-                <MyPickEmptyView pickType="neoga" />
-              )}
-            </StMyNeogaPickList>
-          )
+        {answerInfo?.pages && (
+          <StMyNeogaFormList>
+            <ProfileListSelectable
+              profiles={answerInfo.pages.map((page) => page.formList).flat()}
```

Comment:
```suggestion
              profiles={answerInfo.pages[0]?.formList ?? []}
```

이렇게 하면 임시방편으로 해결될 것이에요